### PR TITLE
Pinpoint

### PR DESCRIPTION
**Pull request by RoySRose**
Hello, @ivansenic 

I also thought that this kind of platform was needed and this looks better than what I've imagined~!

This PR is to add Pinpoint APM on the landscape model.
Pinpoint is widely used APM, specialized for distributed systems. Check our [homepage](http://naver.github.io/pinpoint/index.html) to get to know better.

### CODE REVIEW

File: model/components/pinpoint.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      - Apache License 2.0
+    categories:
+      - agent
+    capabilities:
+      tech:
+        - java
+      aspects:
+        - tracing
+        - metrics [app]
+        - metrics [host]
```
Comment (AlexanderWert) [inline model/components/pinpoint.yml]: Are these really host metrics or do you mean the metrics you get from a JVM?
For example when reading the CPU utilization is the the utilization of the JVM or actually of the host?

If you mean the JVM metrics then change this please to metrics [process].

File: model/components/pinpoint.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+components:
+
+  # Pinpoint Agent
+  - id: pinpoint-agents
```
Comment (AlexanderWert) [inline model/components/pinpoint.yml]: is it on purpose that the id is in plural form "pinpoint-agents" instead of "pinpoint-agent"?

Comment (RoySRose) [reply model/components/pinpoint.yml]: Actually, there isn't any purpose. I've taken a look at what others have done. Few of them that I've looked first was in plural. I'll change them:)

File: model/components/pinpoint.yml (hunk 1 above)
Comment (RoySRose) [reply model/components/pinpoint.yml]: got it:)

**Pull request merged**